src/vivarium_testing_utils/automated_validation/interface.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+import pandas as pd
+from layered_config_tree import LayeredConfigTree
+
+from vivarium_testing_utils.automated_validation import plot_utils
+from vivarium_testing_utils.automated_validation.comparison import Comparison
+from vivarium_testing_utils.automated_validation.data_loader import DataLoader
+
+
+class ValidationContext:
+    def __init__(self, results_dir: str | Path, age_groups: pd.DataFrame | None):
+        self.data_loader = DataLoader(results_dir)
+        self.comparisons = LayeredConfigTree()
+
+    def sim_outputs(self):
+        return self.data_loader.sim_outputs()
+
+    def artifact_keys(self):
```

Comment:
Nit: I like methods to be verbs and attributes/properties to be nouns. We can definitely discuss if we think that ought to be a rule.